### DISCUSSION TITLE
CSS not rendaring in build file in public directory

### DISCUSSION THREAD

**Discussion by Taraldinn** [thuliteio/doks#901] [Q&A]
![image](https://user-images.githubusercontent.com/59772487/190322953-91f4dc99-3b3b-46cc-a279-61a7764c931d.png)
I have created this and i have build this . but after building it's not rendering any css and hot to do this solve ?

**Comment by h-enk** [comment]
Please take a look at https://getdoks.org/docs/reference/security/#content-security-policy + https://getdoks.org/docs/how-to/setup/project-configuration/#postcssconfigjs
